Sources/Protocols/MessagesDataSource.swift

```diff
@@ -51,16 +51,6 @@ public protocol MessagesDataSource: AnyObject {
     ///   - messagesCollectionView: The `MessagesCollectionView` in which the messages will be displayed.
     func numberOfMessages(in messagesCollectionView: MessagesCollectionView) -> Int
 
-    /// The `Avatar` information to be used by the `AvatarView`.
-    ///
-    /// - Parameters:
-    ///   - message: The `MessageType` that will be displayed by this cell.
-    ///   - indexPath: The `IndexPath` of the cell.
-    ///   - messagesCollectionView: The `MessagesCollectionView` in which this cell will be displayed.
-    ///
-    /// The default value returned by this method is `Avatar()`.
-    func avatar(for message: MessageType, at  indexPath: IndexPath, in messagesCollectionView: MessagesCollectionView) -> Avatar
```

Comment:
We have to add this to `MessageKit+Availability.swift` to  let people know this change

Reply:
ok.